src/robot_launch_files/emoticon_handler

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/python
```

Comment:
- Don't we have another package to keep these kind of scripts.
- In anyway, IMHO a script belongs in the scripts directory
- Amigo is hardcoded: stuff is not useful for SERGIO (topics are hardcoded,  Amigo object is loaded)